docs/api-reference.md

```diff
@@ -3,25 +3,623 @@ id: api-reference
+    set(value) {
+      fs.writeFileSync(`${prefix}${readModelName}`, JSON.stringify(value))
+    }
+  }
+  return store
+}
+```
+
+<!-- prettier-ignore-end -->
+
+### disconnect
+
+Closes the storage connection.
+
+#### Arguments
+
+| Argument Name | Description                 |
+| ------------- | --------------------------- |
+| store         | A store object.             |
+| readModelName | A read model to disconnect. |
+
+#### Example
+
+<!-- prettier-ignore-start -->
+
+[mdis]:# (../tests/custom-readmodel-sample/connector.js#disconnect)
+```js
+const disconnect = async (store, readModelName) => {
+  safeUnlinkSync(`${prefix}${readModelName}.lock`)
+  readModels.delete(readModelName)
+}
+```
+
+<!-- prettier-ignore-end -->
+
+### drop
+
+Removes the Read Model's data from storage.
+
+#### Arguments
+
+| Argument Name | Description                        |
+| ------------- | ---------------------------------- |
+| store         | A store object.                    |
+| readModelName | A Read Model whose data to remove. |
+
+#### Example
+
+<!-- prettier-ignore-start -->
+
+[mdis]:# (../tests/custom-readmodel-sample/connector.js#drop)
+```js
+const drop = async (store, readModelName) => {
+  safeUnlinkSync(`${prefix}${readModelName}.lock`)
+  safeUnlinkSync(`${prefix}${readModelName}`)
+}
+```
+
+<!-- prettier-ignore-end -->
+
+### dispose
+
+Forcefully disposes all unmanaged resources that Read Models served by this connector use.
+
+#### Example
+
+<!-- prettier-ignore-start -->
+
+[mdis]:# (../tests/custom-readmodel-sample/connector.js#dispose)
+```js
+const dispose = async () => {
+  for (const readModelName of readModels) {
+    safeUnlinkSync(`${prefix}${readModelName}.lock`)
+  }
+  readModels.clear()
+}
+```
+
+<!-- prettier-ignore-end -->
+
+## Read Model Store Interface
+
+The table below lists functions that you can use to communicate with a Read Model store through a `store` object.
+
+| Function Name               | Description                                                                        |
+| --------------------------- | ---------------------------------------------------------------------------------- |
+| [defineTable](#definetable) | Defines a new table within the store.                                              |
+| [find](#find)               | Searches for data items that based on the specified expression.                    |
+| [findOne](#findone)         | Searches for a single data item based on the specified expression.                 |
+| [count](#count)             | Returns the count of items that meet the specified condition.                      |
+| [insert](#insert)           | Inserts an item into the specified table.                                          |
+| [update](#update)           | Searches for data items and updates them based on the specified update expression. |
+| [delete](#delete)           | Deletes data items based on the specified search expression.                       |
```

Comment:
"the specified expression", "the specified expression", "the specified condition", " the specified table", "the specified update expression", ""the specified search expression"?

Reply:
Removed that 'specified' part from all descriptions in the table